apis/core/v1alpha1/flagsourceconfiguration_types.go

```diff
@@ -95,68 +95,134 @@ type FlagSourceConfigurationSpec struct {
 	// +optional
 	DefaultSyncProvider SyncProviderType `json:"defaultSyncProvider"`
 
+	// Sources defines the syncProviders and associated configuration to be applied to the sidecar
```

Comment:
I think the answer is simply that we use v1 as our hub, only because it was easier/faster. This is a non-breaking change so it doesn't cause a problem AFAIK

For alpha it was the path of least resistance. Eventually to keep things cleaner, when there is less flux in our CRDs, we should probably make beta the hub and updated all the conversions. I don't think that should be done in this PR though.

Reply:
@odubajDT if you're happy to take this approach, continuing along the path of least resistance followed by a 'cleanup' in the beta release then i'm happy to merge this in now

Reply:
Here is the existing research issue for this: https://github.com/open-feature/open-feature-operator/issues/352